metrics/tarantool/luajit.lua

```diff
@@ -34,46 +13,52 @@ local function update()
     -- Details: https://github.com/tarantool/doc/issues/1597
     local lj_metrics = misc.getmetrics()
     collectors_list.gc_freed =
-        set_counter('gc_freed', 'Total amount of freed memory', lj_metrics.gc_freed)
+        utils.set_gauge('gc_freed', 'Total amount of freed memory', lj_metrics.gc_freed, nil, LJ_PREFIX)
```

Comment:
There was some ideas on `reset`/`inc` counter approach for monotonic metrics here: https://github.com/tarantool/metrics/pull/128#discussion_r508520625 . Maybe we should use this approach when https://github.com/tarantool/metrics/pull/288#pullrequestreview-737155916 instead of the current one? For example, when using prometheus type of metric is shown in Grafana UI.